e2e/scenario_test.go

```diff
@@ -15,39 +15,6 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v6"
 )
 
-func Test_AzureLinuxV2(t *testing.T) {
```

Comment:
Remove this test as it's covered by `Test_AzureLinuxV2_KubeletCustomConfig` and save some test time.